### PR TITLE
Docs/add default devs tab

### PR DESCRIPTION
**Pull request by ethanshar**
## Description
**Depends on https://github.com/wix/react-native-ui-lib/pull/3387**

To avoid having the UX adding the same default Devs tab I added it in our ComponentPage component 
I also fixed an issue I found with rendering extending components link 

## Changelog
N/A

### CODE REVIEW

File: docuilib/src/components/pageComponents/DefaultTabs.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    //   type: 'banner',
+    //   title: 'Banner',
+    //   description: 'This is some massage to the user',
+    //   color: 'pink',
+    //   icon: ''
+    // },
+    {
+      type: 'props',
+      title: 'API',
+      description: 'This is the list of additional props for the Text component'
```
Comment (Inbal-Tish) [inline docuilib/src/components/pageComponents/DefaultTabs.ts:29]:
```suggestion
      description: 'This is the list of additional props for the component'
```

File: docuilib/src/components/pageComponents/DefaultTabs.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      title: 'Usage',
+      type: 'usage'
+    },
+    {
+      type: 'info'
+    },
+    {
+      type: 'tip'
+    },
+    // {
```
Comment (Inbal-Tish) [inline docuilib/src/components/pageComponents/DefaultTabs.ts:14]: You can remove the commented code here, this section can be added in noci

File: docuilib/src/components/pageComponents/DefaultTabs.ts (hunk 1 above)
Comment (ethanshar) [reply docuilib/src/components/pageComponents/DefaultTabs.ts]: Fixed manually

File: docuilib/src/components/pageComponents/DefaultTabs.ts (hunk 2 above)
Comment (ethanshar) [reply docuilib/src/components/pageComponents/DefaultTabs.ts]: Fixed

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**